docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -36,9 +36,241 @@ You get ready-to-use code tailored to your needs!
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud({
+		version: "43.0.0",
+		ckbox: {
+			version: "2.5.1",
+			// Optional, by default it's already 'lark':
+			theme: "lark",
+		},
+	});
+
+	if (cloud.status === "error") {
+		return <div>Error!</div>;
+	}
+
+	if (cloud.status === "loading") {
+		return <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor: ClassicEditorBase,
+		Bold,
+		Essentials,
+		Italic,
+		Mention,
+		Paragraph,
+		Undo,
+		CKBox,
+		CKBoxImageEdit,
+	} = cloud.CKEditor;
+
+	return (
+		<CKEditor
+			editor={ClassicEditorBase}
+			data={"<p>Hello world!!!</p>"}
+			config={{
+				toolbar: {
+					items: ["undo", "redo", "|", "bold", "italic"],
+				},
+				plugins: [
+					Bold,
+					Essentials,
+					Italic,
+					Mention,
+					Paragraph,
+					Undo,
+					CKBox,
+					CKBoxImageEdit,
+				],
+				ckbox: {
+					tokenUrl: "https://api.ckbox.io/token/demo",
+					forceDemoLabel: true,
+					allowExternalImagesEditing: [/^data:/, /^i.imgur.com\//, "origin"],
+				},
+			}}
+		/>
+	);
+};
+```
+
+### Usage with external plugins
+
+There are various ways to use external plugins. Here is a list of them:
+
+* **Local UMD Plugins:** Dynamically import local UMD modules using import() syntax.
+* **Local External Imports:** Load external plugins locally using additional bundler configurations (such as Vite).
+* **CDN 3rd Party Plugins:** Load JavaScript and CSS files from a CDN by specifying the URLs.
+* **Verbose Configuration:** Advanced plugin loading with options to specify both script and style sheet URLs, along with an optional `checkPluginLoaded` function to verify the plugin has been correctly loaded into the global scope.
+
+Here is an example:
+
+```js
+import React from 'react';
+import { CKEditor, useCKEditorCloud } from '@ckeditor/ckeditor5-react';
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: '43.0.0',
+		plugins: {
+			// New one, example: local UMD plugins
+			Plugin1: async () => import( './your-local-import.umd.js' ),
+
+			// New one, example: local external imports
+			// Additional bundler configuration must be used, like this one:
+			// https://github.com/crcong/vite-plugin-externals
+			Plugin11: async () => import( './your-local-import' ) as unknown as Promise<Your_Type>,
+
+			// New one, example: CDN 3rd party
+			Plugin2: [
+				'https://cdn.example.com/plugin2.js',
+				'https://cdn.example.com/plugin2.css'
+			],
+
+			// Old one, verbose for more advanced plugins. It's still available.
+			Plugin3: {
+				scripts: [ 'https://cdn.example.com/plugin3.js' ],
+				stylesheets: [ 'https://cdn.example.com/plugin3.css' ],
+
+				// Optional, if it's not passed then the type of `Plugin3` will be picked from `Window`
+				checkPluginLoaded: () => ( window as any ).Plugin3
+			}
```

Comment:
That `new one`, `old one` and `it's still availble` comments should be removed. It was used only for demo purposes